Source/WebCore/editing/cocoa/HTMLConverter.mm

```diff
@@ -480,72 +485,73 @@ static bool stringFromCSSValue(CSSValue& value, String& result)
 
     switch (propertyId) {
     case CSSPropertyDisplay:
-        if (element.hasTagName(headTag) || element.hasTagName(scriptTag) || element.hasTagName(appletTag) || element.hasTagName(noframesTag))
+        if (element->hasTagName(headTag) || element->hasTagName(scriptTag) || element->hasTagName(appletTag) || element->hasTagName(noframesTag))
```

Comment:
Should we make this a `switch(element->nodeName())`? Wasn't sure how much additional refactoring I should throw in. Could also do it as a follow-up.

Reply:
Given the number of checks here, I think switching to a `switch(element->nodeName())` is a no-brainer. Feel free to do in this patch or as a follow-up.